tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -1239,12 +1239,20 @@ impl<'a> field::Visit for DefaultVisitor<'a> {
             return;
         }
 
+        let name = field.name();
+
+        // Skip fields that are actually log metadata that have already been handled
+        #[cfg(feature = "tracing-log")]
+        if name.starts_with("log.") {
+            debug_assert_eq!(self.result, Ok(())); // no need to update self.result
+            return;
+        }
+
+        // emit separating spaces if needed
         self.maybe_pad();
```

Comment:
Just noticed that this updates self.result but it is instantly overwritten… could also change that here?